src/transforms/transform-commit.ts

```diff
@@ -34,10 +40,7 @@ export const transformCommit = (payload): JiraCommitData | undefined => {
 	}
 
 	return {
-		commits: commits,
-		id: payload.repository.id.toString(),
-		name: payload.repository.full_name,
-		url: payload.repository.html_url,
-		updateSequenceId: Date.now()
+		... transformRepositoryDevInfoBulk(payload.repository, gitHubBaseUrl),
```

Comment:
Drag-n-drop replacement (with exception of changing repository.name to repository.full_name - this is what we have in other places), not feature-flagging
<img width="1278" alt="image" src="https://user-images.githubusercontent.com/20631664/194985862-d4892200-3d84-4260-a6cc-67dad9edc5c0.png">
(on the left - removed lines, on the right - the mapping inside the function)